src/librustc_plugin/lib.rs

```diff
@@ -47,8 +47,9 @@
 //! #![plugin(myplugin)]
 //! ```
 //!
-//! See the [`plugin` feature](../unstable-book/language-features/plugin.html) of
-//! the Unstable Book for more examples.
+//! See the [`plugin`
+//! feature](https://doc.rust-lang.org/nightly/unstable-book/language-features/plugin.html)
```

Comment:
Replace the link with relative link: `../../unstable-book/language-features/plugin.html`

Reply:
Thanks @lzutao . I also prefer relative links. But I'm not sure whether `../../unstable-book/language-features/plugin.html` will affect offline documents. (See https://github.com/rust-lang/rust/issues/57489#issuecomment-482933292)

Reply:
What is local doc by the way? Is it distributed by Rust or you have to build
it locally?

Reply:
I guess the doc distributed by Rust doesn't include [rustc internal doc](https://doc.rust-lang.org/nightly/nightly-rustc/rustc/), which is to be fixed in this PR. So I have to build it for a local copy.
The unstable book is distributed with rustup.